resources/sql/competitions.sql

```diff
@@ -1,4 +1,4 @@
--- :name competitions-create! :! :n
```

Comment:
@axrs When making a change to someones PR do I tag them for review? If so is there another way of doing that other than the way I just tagged you now?